nf_core/__main__.py

```diff
@@ -465,7 +466,9 @@ def create_module(ctx, directory, tool, author, label, meta, no_meta, force):
 
     # Run function
     try:
-        module_create = nf_core.modules.ModuleCreate(directory, tool, author, label, has_meta, force)
+        module_create = nf_core.modules.ModuleCreate(
+            directory, tool, author, label, has_meta, force, conda_name=conda_name
```

Comment:
```suggestion
            directory, tool, author, label, has_meta, force, conda_name
```

Reply:
Was wondering why you had `conda_name=conda_name` just for this parameter?

Reply:
It was at the end of the list and I like to specify that if possible 🤷  can remove of course ...